src/main/java/com/github/paweladamski/httpclientmock/matchers/HttpResponseMatchers.java

```diff
@@ -70,14 +66,14 @@ public void describeTo(Description description) {
       }
 
       private String getCookieValue(CookieStore cookieStore, String cookieName) {
-        if (cookieStore != null) {
-          for (Cookie cookie : cookieStore.getCookies()) {
-            if (cookie.getName().equalsIgnoreCase(cookieName)) {
-              return cookie.getValue();
-            }
-          }
+        if (cookieStore == null) {
+          return null;
         }
-        return null;
+
+        Optional<Cookie> cookie = cookieStore.getCookies().stream()
+          .filter(c -> c.getName().equalsIgnoreCase(cookieName))
+        .findFirst();
+        return cookie.isPresent() ? cookie.get().getValue() : null;
```

Comment:
You can write:
```
return cookieStore.getCookies().stream()
          .filter(c -> c.getName().equalsIgnoreCase(cookieName))
        .findFirst()
        .map(cookie->cookie.get().getValue())
        .orElse(null);

Reply:
Fixed in b659b35b08fb18e8e6c362d073ce5021207188c3